<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="7" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="30160" started-at="2021-06-24T07:54:44Z" finished-at="2021-06-24T07:55:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="30160" started-at="2021-06-24T07:54:44Z" finished-at="2021-06-24T07:55:14Z">
      <class name="com.qa.qs.tests.HomePageTest">
        <test-method status="PASS" signature="SetUp()[pri:0, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="SetUp" is-config="true" duration-ms="13502" started-at="2021-06-24T07:54:44Z" finished-at="2021-06-24T07:54:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method status="PASS" signature="homePageHeaderTest()[pri:1, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="homePageHeaderTest" duration-ms="4075" started-at="2021-06-24T07:54:58Z" finished-at="2021-06-24T07:55:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePageHeaderTest -->
        <test-method status="PASS" signature="homePageTitleTest()[pri:2, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="homePageTitleTest" duration-ms="25" started-at="2021-06-24T07:55:02Z" finished-at="2021-06-24T07:55:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePageTitleTest -->
        <test-method status="FAIL" signature="userAccountNameTest()[pri:3, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="userAccountNameTest" duration-ms="3817" started-at="2021-06-24T07:55:02Z" finished-at="2021-06-24T07:55:06Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [DL] but found [SH13]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [DL] but found [SH13]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.qs.tests.HomePageTest.userAccountNameTest(HomePageTest.java:60)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userAccountNameTest -->
        <test-method status="PASS" signature="iconHomeExistsTest()[pri:4, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="iconHomeExistsTest" duration-ms="1929" started-at="2021-06-24T07:55:06Z" finished-at="2021-06-24T07:55:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iconHomeExistsTest -->
        <test-method status="PASS" signature="iconMenuExistsTest()[pri:5, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="iconMenuExistsTest" duration-ms="1910" started-at="2021-06-24T07:55:07Z" finished-at="2021-06-24T07:55:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iconMenuExistsTest -->
        <test-method status="PASS" signature="iconHelpExistsTest()[pri:6, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="iconHelpExistsTest" duration-ms="1929" started-at="2021-06-24T07:55:09Z" finished-at="2021-06-24T07:55:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iconHelpExistsTest -->
        <test-method status="PASS" signature="iconAcountExistsTest()[pri:7, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="iconAcountExistsTest" duration-ms="1922" started-at="2021-06-24T07:55:11Z" finished-at="2021-06-24T07:55:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iconAcountExistsTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.qs.tests.HomePageTest@233fe9b6]" name="tearDown" is-config="true" duration-ms="940" started-at="2021-06-24T07:55:13Z" finished-at="2021-06-24T07:55:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.qs.tests.HomePageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
